test/module/irohad/ordering/ordering_service_test.cpp

```diff
@@ -36,96 +44,140 @@ using ::testing::DoAll;
 using ::testing::AtLeast;
 using ::testing::Return;
 
-class OrderingServiceTest : public OrderingTest {
+static logger::Logger log_ = logger::testLog("OrderingService");
+
+class MockOrderingServiceTransport : public network::OrderingServiceTransport {
  public:
-  OrderingServiceTest() {
-    fake_gate =
-        static_cast<ordering::MockOrderingGate *>(gate_transport_service.get());
+
+  void publishProposal(model::Proposal &&proposal,
+                       const std::vector<std::string> &peers) override {
+    publishProposal(proposal, peers);
+  };
+
+  void subscribe(std::shared_ptr<network::OrderingServiceNotification> subscriber) override {
+    subscriber_ = subscriber;
   }
 
-  void SetUp() override {}
+  MOCK_METHOD2(publishProposal,
+               void(const model::Proposal &proposal,
+                    const std::vector<std::string> &peers));
+
+  ~MockOrderingServiceTransport() override = default;
```

Comment:
What is the purpose to add destructor here?

Reply:
useless, deleted